.github/workflows/sonarcloud.yml

```diff
@@ -18,11 +18,16 @@ jobs:
         if: env.SONAR_TOKEN != null
         with:
           fetch-depth: 0 # Disabling shallow clone is recommended for improving relevancy of reporting
-      - name: Set up Python
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        if: env.SONAR_TOKEN != null
```

Comment:
Can't we add this conditional on the entire workflow, or step 1 which aborts the workflow?

Reply:
Testing...

Reply:
Nope. Workflow level environment variables aren't available in the job: https://github.com/actions/runner/issues/1661.